Assets/Scripts/Gameplay/GameplayObjects/AnimationCallbacks/AnimatorTriggeredSpecialFX.cs

```diff
@@ -1,14 +1,13 @@
 using System;
 using System.Collections;
 using System.Collections.Generic;
-using UnityEngine;
-
-#if UNITY_EDITOR
```

Comment:
mm did you try to build the game? This should fail in builds. CI should fail also because of this